uitools/widgets/Esri/ArcGISRuntime/Toolkit/CoordinateConversion.cpp

```diff
@@ -14,6 +14,7 @@
  *  limitations under the License.
  ******************************************************************************/
 #include "CoordinateConversion.h"
+
```

Comment:
Are there any code changes to this file? There are enough formatting changes where I cannot easily tell.

Reply:
Sorry, qdoc was complaining it couldn't match documentation to methods - running the formatter seemed to fix this issue for whatever reason. No actual changes, just squashing qdoc's warnings with cargo-cultish fixes.

Reply:
Thanks for confirming.